src/main/java/de/rabea/game/GameSetup.java

```diff
@@ -15,17 +15,28 @@ public GameSetup(UserInterface userInterface, PlayerFactory playerFactory) {
 
     public void startApplication() {
         userInterface.greet();
-        setUpNewGame();
+        setUpGame();
     }
 
-    public void setUpNewGame() {
+    public void setUpGame() {
```

Comment:
It seems this method is doing more than just setting up a game.
Maybe splitting the responsibilities or changing the name could improve the intent of that method.